include/ws_core.inc.php

```diff
@@ -401,7 +401,10 @@ function getMethodOptions($methodName)
 
   static function isPost()
   {
-    return isset($HTTP_RAW_POST_DATA) or !empty($_POST);
```

Comment:
I think this can just be `return $_SERVER['REQUEST_METHOD'] === 'POST';`

That still uses a global variable, but at least it's not deprecated.

Note that filter_input() doesn't work with INPUT_SERVER or INPUT_ENV when you use FASTCGI:  https://stackoverflow.com/a/36205923/99667